### ISSUE TITLE
Box is always display: block

### ISSUE THREAD

**Issue by dandelionadia** [kettanaito/atomic-layout#202]
`<Box as={StyledListItem}>` is `display: block` by default. I wrote `display: inline-block` for `<Box as={StyledListItem}>` but it's still  `display: block` . 

```jsx
const StyledListItem = styled.div`
    display: inline-block;
`
```

```jsx
<Box as={StyledListItem} marginBottom={2}>
    ...
 </Box>
```

**Comment by kettanaito** [comment]
Interestingly enough, this specificity has been introduced to fix some CSS assignment problems:

https://github.com/kettanaito/atomic-layout/blob/03dbaa5795c3d9e83939976d44c79922642502ac/src/components/Box.tsx#L12-L24

Now it looks like it blocks custom styled components properties from being applied. This needs a good test.

**Comment by kettanaito** [comment]
The issue is fixed and published under `0.9.3`.
